### DISCUSSION TITLE
Can I import 3d pointcloud (.pcd) file to assist the road drawing like JOSM?

### DISCUSSION THREAD

**Discussion by vincentcheng62** [CommonRoad/commonroad-scenario-designer#12] [Q&A] [answered]
Can I import 3d pointcloud (.pcd) file to assist the road drawing like JOSM?
Thanks!

**Comment by smaierhofer** [comment, marked as answer]
Unfortunately, this is not possible. 
You can only place an aerial image from Bing in the background.
